n/numpy/build_info.json

```diff
@@ -11,5 +11,8 @@
   "use_non_root_user": false,
   "v1.*.*": {
     "build_script": "numpy_ubi_9.3.sh"
+  },
```

Comment:
@kiranNukal - Order should have first exact version then regex pattern.. else it will always pick default